package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Js Project",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tanzila-abedin/restaurant-page.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/tanzila-abedin/restaurant-page/issues"
+  },
+  "homepage": "https://github.com/tanzila-abedin/restaurant-page#readme",
+  "devDependencies": {
+    "webpack": "^5.46.0",
+    "webpack-cli": "^4.7.2"
+  },
```

Comment:
- [ ] It will be great to have all the webpack related dependencies added to your project. This will improve the webpack configuration of your project. It seems you forgot to add one dependency which is `webpack-dev-server`. ⬇️ 
![devdep_requirement](https://user-images.githubusercontent.com/32361963/127466441-8a840157-fda5-497e-9fea-bf7390bb94c9.png)